WordPress/jetpack_metadata/release_notes.txt

```diff
@@ -1,4 +1,7 @@
-- Font size and line height are now supported for block themes.
-- Add new stat cards in the Settings menu on the Stats screen.
-- When changing the media in an Image block that’s also a post’s Featured Image, get a prompt to replace the Featured Image.
-- Share text, images, and video in the app using your device’s Share action.
+* [*] Comment editor: Notification comments support in the new editor [https://github.com/wordpress-mobile/WordPress-Android/pull/15884]
+* [*] Site creation: Fixed bug where sites created within the app were not given the correct time zone, leading to post scheduling issues. [https://github.com/wordpress-mobile/WordPress-Android/pull/15904]
+* [*] Login Epilogue: Fixed bug where if no sites available, then new login epilogue screen without clickable sites and with "create new site" is shown instead of no sites empty "My Site" view. [https://github.com/wordpress-mobile/WordPress-Android/pull/15944]
+* [*] Fixes flickering when changing the preview mode in Page or Site design picker [https://github.com/wordpress-mobile/WordPress-Android/pull/15943]
+* [**] Jetpack App: Signup using WordPress.com, create your favorite site and enjoy the Reader inside Jetpack app. [https://github.com/wordpress-mobile/WordPress-Android/pull/15946]
```

Comment:
(Note that this one was removed from the WordPress notes as it seems to be Jetpack-app-only.)

Reply:
Addressed in [681333f](https://github.com/wordpress-mobile/WordPress-Android/pull/15986/commits/681333ff5bee95e216234470184fd4363154d507)